pkg/v1/providers/ytt/lib/helpers.star

```diff
@@ -328,3 +329,24 @@ def get_labels_map_from_string(labelString):
    end
    return labelMap
 end
+
+def compare_semver_versions(a, b):
```

Comment:
Suggestion: Try to leverage ytt version module https://carvel.dev/ytt/docs/v0.41.0/lang-ref-ytt-version/#docs

Reply:
good to know that, will update accordingly, thanks !

Reply:
sounds like `version.require_at_least` has only one parameter and is used to check ytt version ? try to find other similar function that could compare two versions but seems no luck for me ~